jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Advertisement.java

```diff
@@ -109,7 +110,9 @@ private static StringBuilder createAdvertContent(String vmId, String displayName
 		addKeyValue(contentBuffer, KEY_VM_ID, vmId);
 		addKeyValue(contentBuffer, KEY_DISPLAY_NAME, (((null == displayName) || (displayName.length() == 0))? vmId: displayName));
 		addKeyValue(contentBuffer, KEY_NOTIFIER, CommonDirectory.MASTER_NOTIFIER);
-		
+		/*[IF Sidecar18-SE-OpenJ9]*/
```

Comment:
I thought this was going to be removed, since it is not necessary.

Reply:
Fixed.